python/ray/train/tests/test_minimal.py

```diff
@@ -75,42 +62,6 @@ def train_func():
     assert results.checkpoint.to_dict()[key] == checkpoint.to_dict()[key]
 
 
-def test_run_legacy(ray_start_4_cpus):
```

Comment:
can this be written using DataParallelTrainer? I think it would still be good to check we are not leaking dependencies.

Reply:
ah sorry didn't see that